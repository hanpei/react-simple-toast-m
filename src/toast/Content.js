import React from 'react';
import PropTypes from 'prop-types';

const Content = ({ children, ...props }) => children({ ...props });
Content.propTypes = { children: PropTypes.func.isRequired };

export default Content;