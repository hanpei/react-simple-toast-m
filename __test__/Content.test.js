import React from 'react';
import { shallow } from 'enzyme';
import Content from '../src/toast/Content';

describe('Content', () => {
  it('render', () => {
    const App = <Content>{() => <h1>something test</h1>}</Content>;
    const content = shallow(App);
    expect(content).toMatchSnapshot();
  });
});
