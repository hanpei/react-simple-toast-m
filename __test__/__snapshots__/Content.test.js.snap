// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content render 1`] = `
<h1>
  something test
</h1>
`;
